<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface list</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="nameField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
